arch/arm/core/aarch64/header.S

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <linker/sections.h>
+#include <arch/cpu.h>
+#include "mmu/arm_mmu.h"
+
+#if PAGE_SIZE == 4096
```

Comment:
`PAGE_SIZE` was removed as part of #29551. `CONFIG_MMU_PAGE_SIZE` must be used here.